### DISCUSSION TITLE
selectRaw、DB:raw column disappear.

### DISCUSSION THREAD

**Discussion by sueboy19** [livewire/livewire#7133] [1. Help] [answered]
Livewire 3.0.9
Larvel 10.28.0
PHP 8.2.8
MariaDB 10.4.21

```php
public $orders;

public function query1()
{
    $this->orders = Order::selectRaw('0 as balance')
        ->limit(1)->get();
    info($this->orders);  // Have Datas  [{"balance":0} ... ]
}

pulbic funciton query2()
{
    info($this->orders);   // No Datas  disappear  []
}
```

If use  Order => DB::table('orders'), query2 can get same query1.

Someone know what happen?

**Comment by joshhanley** [comment, marked as answer]
@sueboy19 Livewire doesn't support custom queries, such as group by, selects, etc. when assigning models to properties. So you should use a computed property instead. See here for more details https://livewire.laravel.com/docs/properties#eloquent-constraints-arent-preserved-between-requests

Hope this helps!
